pages/feedback.table.tsx

```diff
@@ -0,0 +1,74 @@
+import 'tailwindcss/tailwind.css';
+import { AppProps } from 'next/app';
+import { useState } from 'react';
+
+function FeedBackTable({ Component, pageProps }: AppProps) {
+  const [values, setValues] = useState({
+    firstName: '', lastName: '', email: '', feedbackText: '' 
+  });
+
+  const set = (name: string) => {
+    return ({ target: { value } }) => {
+      setValues(oldValues => ({ ...oldValues, [name]: value }));
+    }
+  };
+  
+  const saveFormData = async () => {
+    const urlLink = `https://wt-5c9a92f46a5c0d255deef1b49b7375ca-0.run.webtask.io/wallora-feedback/?firstName=${values.firstName}&lastName=${values.lastName}&email=${values.email}&feedback=${values.feedbackText}`
```

Comment:
can we move the url template to env config file? That way we can provide different URL for different env

Reply:
updated